TOPIC8-MONGODB/src/main/java/data/CourseDao.java

```diff
@@ -0,0 +1,15 @@
+package data;
+
+import java.util.List;
+
+import models.Course;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+
+public interface CourseDao extends MongoRepository<Course, Long> {
+
+	@Query(value = "{ $and [ { teacherAssigned.idTeacher: ?0 }, { courseName :?0 }}")

```

Comment:
Please Matias could you explain this query? 
Look this link http://docs.spring.io/spring-data/data-mongo/docs/1.4.0.RC1/reference/html/mongo.repositories.html which says that "Most of the data access operations you usually trigger on a repository result a query being executed against the MongoDB databases. Defining such a query is just a matter of declaring a method on the repository interface"